### ISSUE TITLE
Excalidraw zoom - Extension filter

### ISSUE THREAD

**Issue by javiavid** [beaussan/update-time-on-edit-obsidian#36]
While you are editing in excalidraw and this plugin updates the excalidraw file, the excalidraw zoom changes automatically.

I have seen that it is possible to filter folders in the plugin.
Could you add an option to filter by file extensions?

**Comment by beaussan** [comment]
I can add this feature, however, excalidraw files are stores in .md files so the file filter wouldn't work. Maybe I can add a tag filter to ignore certain tags

**Comment by javiavid** [comment]
It is true, it would be better with the tags, also it can be more useful for other files.

Excalidraw files have this tag by default:
tags: [excalidraw]

**Comment by zsviczian** [comment]
See this thread as well, update-time-on-edit-obsidian causing other issues with Excalidraw as well: https://discord.com/channels/686053708261228577/707816848615407697/1016399748774776864

@beaussan Don't rely on the #excalidraw tag in the front matter. People might take it out. Instead, you can check if a file is an Excalidraw file in two ways:

```typescript
//@ts-ignore
const ea:any = ExcalidrawAutomate; //ea will be undefined if the Excalidraw plugin is not running
const isExcalidrawFile = ea ? ea..isExcalidrawFile(file) : false;
```

or if you want you can add `"obsidian-excalidraw-plugin": "^1.7.11",` to dependencies in package.json, and then use

```typescript
ea = getEA();
const isExcalidrawFile = ea ? ea.isExcalidrawFile(file) : false;
```

**Comment by beaussan** [comment]
I'll add this feature next week then ! Thanks for the snippet, I was looking for a way to detect if it was sa excalidraw file

**Comment by beaussan** [comment]
This should be fixed with the 2.0.1 version :tada:
Feel free to reopen an issue if this is still an issue
